roles/splunk_common/tasks/enable_forwarding.yml

```diff
@@ -16,15 +17,17 @@
   register: setup_indexer_discovery_stanza
   changed_when: setup_indexer_discovery_stanza.status == 201
   no_log: "{{ hide_password }}"
+  notify:
+    - Restart the splunkd service
 
-- name: Setup tcpout:group1 stanza in outputs.conf for clustered deployments
+- name: Setup tcpout group for index-clustering
   uri:
     url: "https://127.0.0.1:{{ splunk.svc_port }}/servicesNS/nobody/system/configs/conf-outputs"
     method: POST
     user: admin
     password: "{{ splunk.password }}"
     validate_certs: false
-    body: "name=tcpout:group1&indexerDiscovery=splunk-indexer&forwardedindex.0.whitelist=_internal&forwardedindex.1.whitelist=_audit&forwardedindex.filter.disable=false"
+    body: "name=tcpout:group1&indexerDiscovery=splunk-indexer"
```

Comment:
FYI - I took the liberty of removing some of the parameters here. I checked in ${SPLUNK_HOME}/etc/system/local and found the defaults for `[tcpout]` can probably be used in lieu of these rules:
```
[tcpout]
maxQueueSize = auto
forwardedindex.0.whitelist = .*
forwardedindex.1.blacklist = _.*
forwardedindex.2.whitelist = (_audit|_internal|_introspection|_telemetry)
forwardedindex.filter.disable = false
```